README.md

```diff
@@ -22,24 +22,63 @@ Check Install
 serverless -v
 ```
 
-Install azure plugin
+Install azure plugins
```

Comment:
Is necessary a specific version of node to install this plugins?

Reply:
No, Only Nodejs should be 10, is due to  azure cli compatibility